src/pages/publication/contributors_tab/components/AddContributorModal.tsx

```diff
@@ -0,0 +1,124 @@
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components';
+
+import { Button, Typography } from '@material-ui/core';
+
+import { getAuthorities } from '../../../../api/authorityApi';
+import SearchBar from '../../../../components/SearchBar';
+import { searchFailure } from '../../../../redux/actions/searchActions';
+import { Authority, emptyAuthority } from '../../../../types/authority.types';
+import { debounce } from '../../../../utils/debounce';
+import AuthorityCard from '../../../user/authority/AuthorityCard';
+
+const StyledClickableDiv = styled.div`
+  cursor: pointer;
+  align-items: center;
+  background-color: ${({ theme }) => theme.palette.box.main};
+  padding-right: 0.5rem;
+`;
+
+const StyledSearchBarContainer = styled.div`
+  width: 35rem;
+`;
+
+const StyledButtonContainer = styled.div`
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 1rem;
+`;
+
+interface DisplayValue {
+  searchTerm: string;
+  results: number;
+}
+
+interface AddContributorModalProps {
+  addAuthor: (selectedAuthor: Authority) => void;
+}
+
+const AddContributorModal: FC<AddContributorModalProps> = ({ addAuthor }) => {
+  const dispatch = useDispatch();
+  const { t } = useTranslation('publication');
+
+  const [matchingAuthorities, setMatchingAuthorities] = useState<Authority[]>();
+  const [selectedAuthor, setSelectedAuthor] = useState<Authority>(emptyAuthority);
+  const [searchTerm, setSearchTerm] = useState('');
+  const [displayValue, setDisplayValue] = useState<DisplayValue>({
+    searchTerm: '',
+    results: 0,
+  });
+
+  const search = useCallback(
+    debounce(async (searchTerm: string) => {
+      const response = await getAuthorities(searchTerm, dispatch);
+      if (response) {
+        setMatchingAuthorities(response);
+        setDisplayValue({ searchTerm, results: response.length });
+      } else {
+        dispatch(searchFailure(t('error.search')));
+      }
+    }),
+    [dispatch, t]
+  );
+
+  const handleSearch = async () => {
+    if (searchTerm.length) {
+      await search(searchTerm);
+    }
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchTerm(event.target.value);
+  };
+
+  return (
+    <>
+      <StyledSearchBarContainer>
+        <SearchBar
+          handleSearch={handleSearch}
+          handleChange={handleChange}
+          searchTerm={searchTerm}
+          resetSearchInput={false}
+        />
+      </StyledSearchBarContainer>
+      {matchingAuthorities && matchingAuthorities.length > 0 ? (
+        <>
+          <Typography variant="h3">
+            {t('profile:authority.search_summary', {
+              searchTerm: displayValue.searchTerm,
+              results: displayValue.results,
+            })}
+          </Typography>
+          {matchingAuthorities.map(authority => (
+            <StyledClickableDiv
+              data-testid="author-radio-button"
```

Comment:
synes ikke vi trenger radio-button og add-knapp. Å klikke på liste-elementet burde holdt.  (første gang skjønte jeg ikke at jeg måtte lete etter en add-knapp)

Reply:
ta det opp med produkteierene 😈

Reply:
we did :-)  og plutselig er alt redesignet...